float64[7] Position
float64[6] Speed
float64[6] Force
float64[6] ArticularPosition
float64[6] ArticularSpeed
